python-bareos/bareos/bsock/lowlevel.py

```diff
@@ -46,17 +46,21 @@
 from bareos.util.password import Password
 import bareos.exceptions
 
-# Try to load the sslpsk module,
-# with implement TLS-PSK (Transport Layer Security - Pre-Shared-Key)
-# on top of the ssl module.
-# If it is not available, we continue anyway,
+# The ssl module support TLS-PSK (Transport Layer Security - Pre-Shared-Key)
+# since Python >= 3.13.
+# For some older Python versions, the TLS-PSK functionality
+# can be added by the sslpsk module,
+# with implement TLS-PSK on top of the ssl module.
+# If it is also not available, we continue anyway,
 # but don't use TLS-PSK.
-try:
-    import sslpsk
-except ImportError:
-    warnings.warn(
-        "Connection encryption via TLS-PSK is not available, as the module sslpsk is not installed."
-    )
+if not getattr(ssl, "HAS_PSK", False):
```

Comment:
Shouldn't we use the positive formulation
if getattr(ssl, "HAS_PSK", False):
    import ....
like it is used line 276, the not here may lead to confusion

Reply:
I'm not sure, how to solve this. The only thing I can think of is:

```
if getattr(ssl, "HAS_PSK", False):
  # The ssl module has support for TLS-PSK.
  pass
else:
  # try to load sslpsk, to add TLS-PSK support to ssl.
  try:
    import sslpsk
    ...
```

Do you think, this is better? I'm not convinced.